.gitignore

```diff
@@ -1,6 +1,7 @@
 /dist
 /ooniprobe
 /coverage.cov
+/ooni.cov
```

Comment:
Why do we have two differently named files for coverage?

Reply:
I suspect `coverage.cov` is how it used to be called at some point in the past. Maybe we should look into dropping that one from the `.gitignore` or maybe just do `.gitignore` for `*.cov`?

Reply:
I am confused by what I see, tho:

```bash
sbs@synchronicity probe-cli % git grep ooni.cov
sbs@synchronicity probe-cli % git grep coverage.cov
.gitignore:/coverage.cov
build.sh:  $0 __docker go test -v -coverprofile=coverage.cov -coverpkg=./... ./...
build.sh:          -coverprofile=coverage.cov -service=travis-ci
build.sh:  go test -v -race -coverprofile=coverage.cov -coverpkg=./... ./...
sbs@synchronicity probe-cli % git branch 
* issue/973
  master
```

Reply:
(`issue/973` is branched off the current `master`)

Reply:
```
probe-cli [fix/ooni_home] % ls -l ooni.cov
-rw-r--r--  1 x  staff  16043 Dec  2 14:17 ooni.cov
````

Hum ok, maybe this was an old file I had lying around from a time prior to some refactor?

Reply:
ah, could be!